examples/doomfire/main.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 The Ebiten Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build example jsgo
+
+package main
+
+import (
+	"image/color"
+	"log"
+	"math/rand"
+	"time"
+
+	"github.com/hajimehoshi/ebiten"
+)
+
+const (
+	screenWidth  = 100
+	screenHeight = 50
+	screenSize   = screenWidth * screenHeight
+)
+
+var (
+	pixels      = make([]byte, screenSize*4)
+	firePixels  = make([]byte, screenSize)
+	firePalette = []color.RGBA{
+		{R: 7, G: 7, B: 7},       //  0
```

Comment:
Add `A: 255`, or the color values don't make sense. I understand alpha values are not used in this example, but let's avoid confusion.

Reply:
OK! As an example, it is best to be as clear as possible.